templates/vue/src/components/tyde/activities/CircleOfSupport/CircleView/index.vue

```diff
@@ -73,6 +73,28 @@
         src="https://twemoji.maxcdn.com/v/13.1.0/72x72/1f60a.png"
       />
     </div>
+    <div>
```

Comment:
Not sure why this `<div>` wrapper is needed

Reply:
removed